beacon_node/beacon_chain/src/canonical_head.rs

```diff
@@ -966,6 +972,9 @@ impl<T: BeaconChainTypes> BeaconChain<T> {
             self.head_tracker.clone(),
         )?;
 
+        // Take a write-lock on the canonical head and signal for it to prune.
+        self.canonical_head.fork_choice_write_lock.prune()?;
```

Comment:
We need parens here I think

```suggestion
        self.canonical_head.fork_choice_write_lock().prune()?;
```